airlock/templates/requests_workspace.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+  {% #card title="All requests in workspace "|add:workspace %}
```

Comment:
Stray space before the second `"`?

Reply:
no, that's correct - if we remove the space we get output like `All requests in workspaceexample-workspace2`